boards/obc/obc.dts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2025 The FINCH CubeSat Project Flight Software Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <st/g4/stm32g431Xb.dtsi>
+
+/ {
+	model = "finch,obc";
+	compatible = "finch,obc";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+		act_led: led_0 {
+			gpios = <&gpioc 2 GPIO_ACTIVE_HIGH>;
+			label = "LED0";
+		};
+		wrn_led: led_1 {
+			gpios = <&gpioc 3 GPIO_ACTIVE_HIGH>;
+			label = "LED1";
+		};
+	};
+
+	aliases {
+		led0 = &act_led;
+		actled = &act_led;
+		led1 = &wrn_led;
+		wrnled = &wrn_led;
+	};
+};
+
+&clk_hse {
+	clock-frequency = <DT_FREQ_M(24)>;
+	status = "okay";
+};
+
+&pll {
+	div-m = <12>;
```

Comment:
@alexapostolu pointed out that this might be wrong.

Reply:
Relevant Reference: RM0440 Rev 9 page 275
https://www.st.com/resource/en/reference_manual/rm0440-stm32g4-series-advanced-armbased-32bit-mcus-stmicroelectronics.pdf

Reply:
`div-m = <6>` should be used along with the other existing PLL configurations to generate a 170 MHz SYSCLK.